checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -335,6 +339,21 @@ def _create_edge(self, origin_vertex_index: int, dest_vertex_index: int, label:
             self.out_edges[origin_vertex_index].append(edge)
             self.in_edges[dest_vertex_index].append(edge)
 
+    def calculate_encryption_attribute(self) -> None:
+        for vertex_index in self.vertices_by_block_type.get(BlockType.RESOURCE, []):
```

Comment:
@achiar99 I think we would benefit from refactoring this method to the base `LocalGraph` class, since its logic is the same in terraform and cloudformation.

Reply:
@rotemavni pls review the next commits